modules/store.js

```diff
@@ -0,0 +1,24 @@
+export default class Store {
+    getBooks = () => {
+      let book;
+      if (localStorage.getItem('bookListss') === null) {
+        book = [];
+      } else {
+        book = JSON.parse(localStorage.getItem('bookListss'));
+      }
```

Comment:
[OPTIONAL]
- [ ] It's better if you write these lines like this `const book = JSON.parse(localStorage.getItem('bookListss')) || [];` it would make it cleaner and more optimized 👍 
- [ ] You can also put the list of books in the `constructor`, this way you won't need to call this method everytime you need the list of books you can just do `this.book` 👍

Reply:
I think since the book variable always needs update there will be no way that it is going to be a constant variable.